functions-and-operators/string-functions.md

```diff
@@ -406,12 +406,63 @@ Output:
 
 ### [`ELT()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_elt)
 
-Return string at index number.
+Returns the element at the index number.
+
+```sql
+SELECT ELT(3, 'This', 'is', 'TiDB');
+```
+
+```
++------------------------------+
+| ELT(3, 'This', 'is', 'TiDB') |
++------------------------------+
+| TiDB                         |
++------------------------------+
+1 row in set (0.00 sec)
+```
+
+This example returns the third element, which is "TiDB".
 
 ### [`EXPORT_SET()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_export-set)
 
 Return a string such that for every bit set in the value bits, you get an on string and for every unset bit, you get an off string.
 
+The full signature of this function is:
+
+```
+EXPORT_SET(bits, on, off, [separator, number_of_bits])
```

Comment:
Extended suggestion:

If `number_of_bits` is not set, it will use 64 as default (the max size of `bits`, which is treated as an unsigned 64 bit integer).
`EXPORT_SET()` will return `number_of_bits` `on`/`off` values optionally separated by `separator`, starting with the right most (lowest) bit as the first (right most) value.

```mysql
tidb> select export_set(b'101',"ON",'off','|',5);
+-------------------------------------+
| export_set(b'101',"ON",'off','|',5) |
+-------------------------------------+
| ON|off|ON|off|off                   |
+-------------------------------------+
1 row in set (0.00 sec)
```